KedarDhere/FlowMagic-SDK
# Make all public

# Proposed Changes
Converted all package elements to public access and relocated `ContentView` to the Tests folder.

# Jira Stories

- [FM-10](https://oregonstate-innovationlab.atlassian.net/browse/FM-10?atlOrigin=eyJpIjoiMWIxNTNhODdkOGNlNDU4ZDg1ZjlhMDA1NjljOTBiMDMiLCJwIjoiaiJ9)

# Checklist

[x] This has been tested  
[x] Self-documenting code  
[x] Code is linted

# Instructions to review

1. Please review all elements in the package for public accessibility.
2. Check the relocation of `ContentView` to the Tests folder.
3. Ensure all unit tests are still running as expected with the moved `ContentView`.
4. Provide feedback on any possible issues or improvements required.

**Testing Instructions:**
To validate the changes:
1. Create a new App in Xcode.
2. Navigate to `File` -> `Add Package`.
3. Provide the GitHub URL: https://github.com/KedarDhere/FlowMagic-SDK.git
4. Add the branch name as `make-all-public`.
5. Verify if everything in the package is accessible for use within the app.
